packages/labextension/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "jupyterlab-jupytext",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
I'd prefer version 1.2.0, because this is a breaking change (i.e. the new extension won't work with JupyterLab 1.x).